Sources/SyrupCore/Generator/Renderer.swift

```diff
@@ -133,7 +133,8 @@ open class Renderer {
 		for enumType in intermediateRepresentation.referencedEnums {
 			let context: [String: Any] = [
 				"moduleName": config.project.moduleName,
-				"enumType": enumType
+				"enumType": enumType,
+				"sortedEnumValues": enumType.values.sorted { lhs, rhs in lhs.value < rhs.value }
```

Comment:
https://github.com/Shopify/syrup/blob/e64905f27f126cb4c1780d951f5f026d4b3610fe/Sources/SyrupCore/GraphQL/IntermediateRepresentationVisitor.swift#L331-L340

If we sort it here we won't have to make changes to the template, and it fixes the problem for both Swift and Kotlin code generation.